compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -255,6 +255,9 @@ error.return.value.too.many=\
 error.return.value.not.enough=\
   not enough return values
 
+error.invalid.client.resource.return.type=\
+  Client resource access does not allow ''{0}'' as return type
```

Comment:
`return type of the resource function does not allow ''{0}'' types`

Reply:
Updated 2504be25de6173179816f9cda84efa8d7ab3883f